src/core/syscalls/syscall_handler.rs

```diff
@@ -580,20 +598,168 @@ mod tests {
 
         let hint_data = HintProcessorData::new_default(GET_TX_INFO.to_string(), ids_data);
         // invoke syscall
-        let mut hint_procesor = SyscallHintProcessor::new_empty().unwrap();
-        let err = hint_procesor.execute_hint(
+        let mut syscall_handler_hint_processor = SyscallHintProcessor::new_empty().unwrap();
+
+        let tx_execution_context = TransactionExecutionContext {
+            n_emitted_events: 50,
+            version: 51,
+            account_contract_address: bigint!(260),
+            max_fee: bigint!(261),
```

Comment:
I'm somehow skeptic that anybody will pay more than `2⁶⁴-1` (in any unit I can think of).